components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodClientProjectSessionTracker.kt

```diff
@@ -141,7 +137,7 @@ class GitpodClientProjectSessionTracker(
                             override fun onNext(ps: Status.PortsStatusResponse) {
                                 for (port in ps.portsList) {
                                     // Avoiding undesired notifications
-                                    if (ignorePorts.contains(port.localPort)) {
+                                    if (ignoredPortsForNotificationService.getIgnoredPorts().contains(port.localPort)) {
```

Comment:
The ignore ports functionality seems to be overcomplicated now, instead of being 3 lines of code, we have 3 files. Am I missing any benefit from doing so?

Reply:
We need one version for 2022.2 API and another for 2022.3.

Or we can remove the ignored ports part.

Reply:
oh, I see :(